src/components/views/rooms/SearchResultTile.js

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2015 OpenMarket Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+'use strict';
+
+var React = require('react');
+var sdk = require('../../../index');
+
+module.exports = React.createClass({
+    displayName: 'SearchResult',
+
+    propTypes: {
+        searchResult: React.PropTypes.object.isRequired,
+        searchHighlights: React.PropTypes.array,
+        onClick: React.PropTypes.func,
```

Comment:
yeah; it wasn't clear that in the long-term, the correct UI was for the user to have to click on the highlighted term. You could envisage a special 'zoom to context' button. So I tried to find something agnostic so that it could be changed later without changing the clients. onClick probably isn't much better. onSelect, maybe?